### PR TITLE
40 about uslocationmap

### PR DESCRIPTION
**Pull request by Klumpish**
# Description:
I have - added a map image and some about us text

Issue #40   

# Type of change:
- [x] create a map image
- [x] add the map
- [x]  add small discription
- [x] style the text and image so that they are next to each other

# Additional information (screenshot, video, code examples):
<details>
<summary>
Image
</summary>

<p>

</p>
</details>
image

### CODE REVIEW

File: src/styles/_features.scss
Comment (jennymakki) [inline src/styles/_features.scss:1]: Odd that it looks like everything has been removed and then added, but to me it looks like the same code so it should be fine.

**Review by jennymakki — APPROVED** [review]

File: src/styles/styles.scss
Comment (Paajt) [inline src/styles/styles.scss:1]: Nice, now it's more consistent 👍

File: src/styles/_footer.scss
Comment (Paajt) [inline src/styles/_footer.scss:1]: Probably the same formatting thing I had on another PR.

File: src/styles/_features.scss
Comment (Paajt) [reply src/styles/_features.scss:1]: I also had the scss-files reformatted for some reason 🤔

**Review by Paajt — APPROVED** [review]
LG2M! Good job! 👍

**Pull request merged**